spec/view_objects/allocations_view_spec.rb

```diff
@@ -132,4 +132,187 @@
       it { is_expected.to be_empty }
     end
   end
+
+  describe "when allocation period is closed" do
```

Comment:
Should this be a context, and regardless of whether it is or isn't it seems like the concept of `allocation journey` is being introduced here. Wonder if the rest of the specs need to be structured in this way e.g (allocation period is open etc) for readability? Not a blocker for merging but may be a refactoring card for the ideas board.

Reply:
Sau bet me again!